### PR TITLE
Add cascade for build config deletions

### PR DESCRIPTION
**Pull request by jimmidyson**
Ref: https://bugzilla.redhat.com/show_bug.cgi?id=1351129

### CODE REVIEW

**Comment by jimmidyson** [comment]
[merge]

File: openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/BuildConfigOperationsImpl.java [hunk 1]
```diff
@@ -131,4 +133,21 @@ public Void trigger(WebHookTrigger trigger) {
 
     return buildConfigOperations;
   }
+
+  private void deleteBuilds() {
+    new BuildOperationsImpl(client, (OpenShiftConfig) config, namespace).withLabel("buildconfig", "edam").delete();
```
Comment (bparees) [inline openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/BuildConfigOperationsImpl.java]: what's this label/value?

Comment (jimmidyson) [inline openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/BuildConfigOperationsImpl.java]: Hahahaha that is awful, that's what that is.

Comment (jimmidyson) [inline openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/BuildConfigOperationsImpl.java]: This was local example code that I've committed by mistake... D'oh!

**Pull request merged**